src/components/Button/ButtonAddPostTopic.js

```diff
@@ -32,6 +35,12 @@ const ButtonAddPostTopic = ({topicName, onRefresh}) => {
   );
 };
 
+ButtonAddPostTopic.propTypes = {
+  topicName: PropTypes.string,
+  onRefresh: PropTypes.func,
+  followType: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The PropTypes have been updated to include `followType`. This is good for type checking in development mode. However, consider adding `.isRequired` to make sure these props are always provided, or provide default values using `defaultProps` if they can be optional.
```diff
  topicName: PropTypes.string,
  onRefresh: PropTypes.func,
+ followType: PropTypes.string
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->